.teamcity/src/main/kotlin/configurations/FlakyTestQuarantine.kt

```diff
@@ -53,4 +54,8 @@ class FlakyTestQuarantine(model: CIBuildModel, stage: Stage, os: Os) : BaseGradl
     steps {
         checkCleanM2AndAndroidUserHome(os)
     }
+
+    dependencies {
```

Comment:
There is an `applyDefaultDependencies()` method which we should use instead here.